src/schemas.ts

```diff
@@ -19,6 +19,15 @@ export type ModelParameter = {
   stream?: boolean
   jsonmode?: boolean
   seed?: number | null
+  tool_choice?:
+    | {
+        type: "function"
+        function: {
+          name: string
+        }
+      }
+    | "auto"
+    | "none"
```

Comment:
`required` is missing https://platform.openai.com/docs/api-reference/chat/create#chat-create-tool_choice

Reply:
added now